### PR TITLE
Adding ruby 2.3.0 to the rbenv available options.

### PR DESCRIPTION
**Pull request by deanwilson**
Upgrade requested in https://trello.com/c/Q40TmMvX

### CODE REVIEW

File: modules/ci_environment/manifests/base.pp [hunk 1]
```diff
@@ -90,6 +90,13 @@
     to_version => '2.2.3'
   }
 
+  rbenv::version { '2.3.0':
+    bundler_version => '1.11.2',
+  }
+  rbenv::alias { '2.3':
+    to_version => '2.3.0'
```
Comment (benlovell) [inline modules/ci_environment/manifests/base.pp]: Is this missing a trailing comma?

**Comment by deanwilson** [comment]
In theory, yes it is missing one, but the house style doesn't have it in that file for aliases which is why i left it off. I'll follow up with another PR later to include the puppet-lint check that'll flag this and fix them up alongside it.

**Comment by benlovell** [comment]
@deanwilson :+1:

**Comment by deanwilson** [comment]
Addressed in https://github.com/alphagov/ci-puppet/pull/374

**Pull request merged**